setup.py

```diff
@@ -45,7 +45,7 @@ def read(path, encoding="utf-8"):
     "sqlalchemy-utils>=0.36.6",
     "antlr4-python3-runtime==4.8",
     "lark>=1.0.0",
-    "pyyaml>=5.1,<5.2",
```

Comment:
Any idea on why this constraint was there earlier?

Reply:
No